src/lib/createElement.js

```diff
@@ -1,5 +1,46 @@
 import { addEvent } from "./eventManager";
 
-export function createElement(vNode) {}
+export function createElement(vNode) {
+  if (vNode === null || vNode === undefined || typeof vNode === "boolean") {
+    return document.createTextNode("");
+  }
+  if (typeof vNode === "string" || typeof vNode === "number") {
+    return document.createTextNode(vNode);
+  }
 
-function updateAttributes($el, props) {}
+  if (Array.isArray(vNode)) {
+    const fragment = document.createDocumentFragment();
+    vNode.flat().forEach((child) => {
+      fragment.appendChild(createElement(child));
+    });
+    return fragment;
```

Comment:
이미 여기서 return을 해서 함수를 종료했으니, 아래 else로 구분하는 것보단 따로 빼는게 가독성있을 것 같아요~
```suggestion
const el = document.createElement(vNode.type);
const props = vNode.props || {};
const children = props.children;
if (children !== undefined && children !== null) {
  const childArray = Array.isArray(children) ? children : [children];
  childArray.forEach((child) => {
    el.appendChild(createElement(child));
  });
}
return el;
```